tuf/tuf.go

```diff
@@ -49,6 +49,20 @@ func (err ErrNotLoaded) Error() string {
 	return fmt.Sprintf("%s role has not been loaded", err.role)
 }
 
+// ErrStopWalk - used by visitor functions to signal WalkTargets to stop walking
+type ErrStopWalk struct{}
```

Comment:
We can assess when we're closer to merging. It may become obvious as we add the additional uses of the walker whether this is necessary or not.